### PR TITLE
Icecast 2.4.4 package including ezstream as client

### PR DESCRIPTION
**Pull request by DigitalBox98**
_Motivation: Move Icecast package to latest 2.4.4 version and include ezstream client
Wizard added for login/password configuration at package installation :)

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

File: cross/ezstream/PLIST [hunk 1]
```diff
@@ -0,0 +1,13 @@
+bin:bin/ezstream
+rsc:bin/ezstream-file.sh
+shr:share/examples/ezstream/ezstream-file_template.xml
+shr:share/examples/ezstream/ezstream_metadata.xml
```
Comment (cytec) [inline cross/ezstream/PLIST]: afaik we don't have `shr` type that should be `rsc`

Comment (digitalbox94) [reply cross/ezstream/PLIST]: Ok noted, I will replace sur by rsc

**Comment by DigitalBox98** [comment]
Ok fixed. 

Regarding squash commits into a single one, not sure how to proceed, but I'm interested on how to do if it's not a big deal :)

**Comment by cytec** [comment]
it's actually pretty straight forward: https://github.com/wprig/wprig/wiki/How-to-squash-commits

**Comment by DigitalBox98** [comment]
Indeed quite straight forward it's done :)

**Comment by cytec** [comment]
@SynoCommunity/developers LGTM

**Comment by Safihre** [comment]
:+1: didn't test it though.

File: spk/icecast/Makefile [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
 
-INSTALLER_SCRIPT = src/installer.sh
-SSS_SCRIPT       = src/dsm-control.sh
-FWPORTS          = src/${SPK_NAME}.sc
-CONF_DIR         = src/conf/
+WIZARDS_DIR = WIZARD_UIFILES
+
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+SERVICE_PORT = 8000
```
Comment (ymartin59) [inline spk/icecast/Makefile]: I documented port at https://github.com/SynoCommunity/spksrc/wiki/SynoCommunity-Used-Ports

File: spk/icecast/Makefile [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
+SERVICE_PORT_TITLE = $(DISPLAY_NAME)
+NO_SERVICE_SHORTCUT = yes
+FWPORTS = src/icecast.sc
 
 INSTALL_PREFIX = /usr/local/$(SPK_NAME)
```
Comment (ymartin59) [inline spk/icecast/Makefile]: Please remove this line, now included in build framework itself.
```suggestion
```

File: spk/icecast/Makefile [hunk 4]
```diff
@@ -1,26 +1,30 @@
...
-FWPORTS          = src/${SPK_NAME}.sc
-CONF_DIR         = src/conf/
+WIZARDS_DIR = WIZARD_UIFILES
+
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+SERVICE_PORT = 8000
+SERVICE_PORT_TITLE = $(DISPLAY_NAME)
+NO_SERVICE_SHORTCUT = yes
+FWPORTS = src/icecast.sc
```
Comment (ymartin59) [inline spk/icecast/Makefile]: I propose to remove that file and variable, as far as it looks like build framework will generate same content by default.

**Comment by DigitalBox98** [comment]
I've tried to squash the last commit on Makefile but unable to.
I will recreate the fork :/

**Comment by DigitalBox98** [comment]
This PR can be removed, I have created a new cleaner one with all the fixes :)

**Comment by hgy59** [comment]
recreated as #3805

**Pull request closed without merging**